pysparkling/rdd.py

```diff
@@ -167,9 +167,9 @@ def seqFuncByKey(tc, i):
                 r[k] = seqFunc(r[k], v)
             return r
 
-        def combFuncByKey(l):
+        def combFuncByKey(lst):
```

Comment:
To disentangle these PRs and to move forward, I'd suggest not to mix issues and not convert `l` to `lst`. I am against abbreviations in variable names and for some of your replacements `l` is not actually a list so we would have to have a discussion about those names which seems to be out of scope for this PR.

Reply:
ok